packages/api-client/src/components/CommandPalette/CommandPaletteImport.vue

```diff
@@ -123,16 +123,28 @@ async function importCollection() {
         :lang="documentType" />
     </template>
     <template #options>
-      <ScalarButton
-        class="p-2 max-h-8 gap-1.5 text-xs hover:bg-b-2 relative"
-        variant="outlined"
-        @click="openSpecFileDialog">
-        JSON, or YAML File
-        <ScalarIcon
-          class="text-c-3"
-          icon="UploadSimple"
-          size="md" />
-      </ScalarButton>
+      <div class="flex flex-row items-center justify-start gap-3">
```

Comment:
This needs to be removed before merge

Reply:
Why do we remove the UI? I'd just enable it by default and keep it. It communicates what happens behind the scenes (super important) and gives the user a little bit more control.

Reply:
oh so anythony/cam can implement something much nicer ✨